src/pages/products/index.jsx

```diff
@@ -156,6 +157,7 @@ const ProductIndex = () => {
                 url={"/products/dau"}
                 publicationDate={"13-11-2020"}
                 blog_slug={"/blog/tags/project/kosh"}
+                updates_slug={"/updates/tags/project/kosh"}
```

Comment:
for both blog_slug and updates_slug make it /dau and not /kosh

Reply:
Sure, let me do that

Reply:
It appears that the /dau pages don't exist. Even in the Tags too, there is no dau tag, so the page for that project tag also doesn't exist. Also as all the project tags are being created based on the project field entry of all the pages ( blogs as well as updates), I think none of the mdx files has dau as a project entry.

![image](https://github.com/user-attachments/assets/bd501f57-7841-43d6-8695-76603c3a9196)

These are the project tags for blogs, they don't have dau tag in them.

Reply:
@maanasb01 its ok if they don't exist. in that case remove the "Read blogs" and "Read updates" section for DAU. will add them later after we've added blog and updates for DAU.

Reply:
Sure!